metabase/metabase
# Fix home and auth backgrounds

Changes:
- Fixes the background image for auth pages
- Fixes the background color for home and auth pages when whitelabeled

How to test:
- Run Metabase EE 
- Make sure auth and home pages are branded
- Enable whitelabeling
- Make sure auth and home pages are neutral

**Default**
<img width="1728" alt="Screenshot 2022-04-22 at 21 25 20" src="https://user-images.githubusercontent.com/8542534/164773421-8b78b166-2da7-4b15-aff2-3c818b09861b.png">
<img width="1728" alt="Screenshot 2022-04-22 at 21 25 40" src="https://user-images.githubusercontent.com/8542534/164773435-b28f2c2c-4aef-454c-85cf-670203ef1bd7.png">

**Whitelabeled**
<img width="1728" alt="Screenshot 2022-04-22 at 21 28 21" src="https://user-images.githubusercontent.com/8542534/164773462-08b3e725-7e4d-462e-857b-f5a9d48fc569.png">
<img width="1728" alt="Screenshot 2022-04-22 at 21 28 33" src="https://user-images.githubusercontent.com/8542534/164773476-57108616-a203-47fe-a707-b71f820bbd3b.png">
